dotCMS/src/main/java/com/dotcms/contenttype/util/KeyValueFieldUtil.java

```diff
@@ -28,8 +28,16 @@
 				return keyValueMap;
 			}
 
+			// the following code fixes issue 10529
+			String json2;

```

Comment:
I like the util suggestion. @jdotcms we will contact you tomorrow about the MarshalUtil